docs/topics/ref_rhsso-deployment-configuration.adoc

```diff
@@ -29,7 +29,7 @@ a|
 The configured **clients accept any callback url**.
 
 a|
-To avoid having a custom configuration per runtime we avoid the verification of the callback as mandated by the OAuth2 specification.
+To avoid having a custom configuration per runtime we avoid the verification of the callback as required by the OAuth2 specification.
```

Comment:
To avoid custom configurations for each runtime, we avoid the callback verification that is required by the OAuth2 specification.

Reply:
Will fix.